src/librustc_mir/transform/nll.rs

```diff
@@ -29,15 +31,17 @@ impl<'a, 'gcx, 'tcx> NLLVisitor<'a, 'gcx, 'tcx> {
         NLLVisitor {
             infcx,
             lookup_map: HashMap::new(),
+            regions: vec![],
         }
     }
 
     pub fn into_results(self) -> HashMap<RegionVid, Lookup> {
```

Comment:
Should we output this as a tuple, i.e. `(HashMap<RegionVid, Lookup>, Vec<Region>)`? Or do we only worry about that when we need to start using the output?